tools/amazon_agent_settings/agent_gemfile

```diff
@@ -0,0 +1,7 @@
+source "https://rubygems.org"
+gem "manageiq-gems-pending", ">0", :require => "manageiq-gems-pending", :git => "https://github.com/ManageIQ/manageiq-gems-pending.git", :branch => "master"
+gem "manageiq-smartstate", "~>0.1.5", :require => "manageiq-smartstate"
+gem "amazon_ssa_support", ">0", :require => "amazon_ssa_support", :git => "https://github.com/ManageIQ/amazon_ssa_support.git", :branch => "master"
```

Comment:
We may want the override_gem thing copied over here, so that we can deal with these git repos on upstream branches.  Note to self, I need to update the branching document to deal with this as well.